Source/SequenceExtensions.cs

```diff
@@ -25,5 +22,27 @@ public static class SequenceExtensions
 		{
 			return new SetupSequentialContext<TMock, TResult>(mock, expression);
 		}
+
+		/// <summary>
+		/// Return a sequence of tasks, once per call.
+		/// </summary>
+		public static ISetupSequentialResult<Task<TResult>> ReturnsAsync<TResult>(this ISetupSequentialResult<Task<TResult>> setup, TResult value)
+		{
+			var tcs = new TaskCompletionSource<TResult>();
```

Comment:
Yes, I was wondering myself too.
Apparently it's available only in .NET 4.5 http://stackoverflow.com/a/21258011/41956
But all projects seems are targeting .NET 4.0 so I decided do not change much  including tests same PR.